v2/concordium/health.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
 package concordium.health;
+option java_package = "com.concordium.grpc.v2";
+option java_multiple_files = true;
```

Comment:
Please provide a small comment explaining what each of the actions does.

Reply:
added comments